server/core/region.go

```diff
@@ -37,6 +38,8 @@ var errRegionIsStale = func(region *metapb.Region, origin *metapb.Region) error
 // RegionInfo records detail region info.
 // Read-Only once created.
 type RegionInfo struct {
+	mu                sync.Mutex
+	countInAntiScore  bool
```

Comment:
Since the only role this `sync.Mutex` plays here is to lock the `countInAntiScore`, what about using `atomic.Value` here?